src/main/java/com/kainos/ea/DTO.java

```diff
@@ -449,4 +449,33 @@ public static boolean isSessionKeyInDB(String sessionKey) throws IOException, SQ
         preparedStmt.execute();
         return Response.Status.OK;
     }
+
+    /**
+     * Get a list of json objects where each JSON object holds the <code>trainingName</code> and the <code>trainingLink</code>.
+     * @return A JSONArray of JSONObjects detailing training information for a specific band.
+     * @param bandName The band level for which you want to view training.
+     * @throws IOException  Create connection to database.
+     * @throws SQLException Invalid SQL syntax.
+     */
+    public static JSONArray getTrainingDataForBand(String bandName) throws IOException, SQLException {
+        Connection c = DBConnector.getConnection();
+
+        PreparedStatement st = c.prepareStatement(
+                "SELECT trainingName, trainingLink FROM KainosSprint.Training \n" +
```

Comment:
Can you just use <space key> instead of \n please. SQL queries can be executed as one line which is the convention we have used everywhere else